build/docker/production.Dockerfile

```diff
@@ -101,7 +102,9 @@ COPY --from=production-build /src/bin/ferretdb /ferretdb
 FROM scratch AS production
 
 COPY --from=production-build /src/bin/ferretdb /ferretdb
+COPY --from=production-build /etc/passwd /etc/passwd
```

Comment:
That probably should work, but that file will contain non-existing users, an invalid home directory for the "ferretdb" user, etc.

I think we better have a static file with a single user  in `build/docker/passwd` and copy it there

Reply:
Well, I think both home directory and shell could be replaced with `/nonexistent`

Reply:
I've created the file.
https://github.com/FerretDB/FerretDB/pull/4004/commits/1798e9bde06189bb66a2c13bc96794c75a651438